### PR TITLE
fix terrafrom config bug

### PR DESCRIPTION
**Pull request by zhongjiajie**

### CODE REVIEW

File: deploy/terraform/aws/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 Set necessary variables by creating a file `ds-ami.pkrvars.hcl` and adding the following variables according to your own usage.
 
-```hcl
+```shel
```
Comment (Radeity) [inline deploy/terraform/aws/README.md:10]: Why not keep hcl?

**Review by Radeity — DISMISSED** [review]
Others LGTM.

File: deploy/terraform/aws/README.md (hunk 1 above)
Comment (zhongjiajie) [reply deploy/terraform/aws/README.md:10]: keep hcl is ok to me, but we execute this command into the shell instead of terrafrom. so I change to shell.

Comment (zhongjiajie) [reply deploy/terraform/aws/README.md:10]: so, do you prefer to use `hcl` or `shell`, all look go for me

Comment (Radeity) [reply deploy/terraform/aws/README.md:10]: Oh, both is okay on me, either.

**Review by Radeity — APPROVED** [review]

**Pull request merged**